### PR TITLE
fix(composer): add new translations

### PR DESCRIPTION
**Pull request by sheilaXu**
## Overview
add new translations

### Scene Composer
For `scene-composer` package changes specifically, you can preview the component in the published storybook artifact. To do this, wait for the `Publish Storybook` action to complete below.

- Click on the workflow details
- Select the Summary item on the left
- Download the zip file

To run the storybook build locally, you need a local static web server:

```
npm install -g httpserver
cd <Extracted Zip Directory>
httpserver
```

Then open the website http://localhost:8080 to run the doc site.

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json [hunk 1]
```diff
@@ -459,6 +531,14 @@
     "note": "label",
     "text": "Select colour"
   },
+  "TZDmnV": {
+    "note": "label for when the user searches for a nonexistent animation",
+    "text": "No matches found"
+  },
+  "TaxKIJ": {
+    "note": "Menu Item label",
+    "text": "Remove all data binding"
```
Comment (mukeshsahay) [inline packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json:540]: It should be `Remove all data bindings`.

**Review by mumanity — APPROVED** [review]

File: packages/scene-composer/translations/IotAppKitSceneComposer.de_DE.json [hunk 2]
```diff
@@ -459,6 +531,14 @@
     "note": "label",
     "text": "Farbe wählen"
   },
+  "TZDmnV": {
+    "note": "label for when the user searches for a nonexistent animation",
```
Comment (divya-sea) [inline packages/scene-composer/translations/IotAppKitSceneComposer.de_DE.json:535]: Maybe we need to start with `capitol` letter?

File: packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json [hunk 3]
```diff
@@ -395,6 +463,10 @@
     "note": "Label clarifying the exact text a user is searching for.",
     "text": "Use: {value}"
   },
+  "PLTmIg": {
+    "note": "circle picker",
+    "text": "circlePicker"
```
Comment (mukeshsahay) [inline packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json:468]: Please check it's usage.

File: packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json [hunk 4]
```diff
@@ -283,6 +339,14 @@
     "note": "Specifies the localized string that describes an option as being selected. This is required to provide a good screen reader experience.",
     "text": "Selected"
   },
+  "H+OF2y": {
+    "note": "Specifies the animation thats being selected",
+    "text": "Autosuggest example with suggestions"
```
Comment (mukeshsahay) [inline packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json:344]: `Auto-suggest`

File: packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json [hunk 5]
```diff
@@ -219,6 +263,14 @@
     "note": "Placeholder",
     "text": "Choose a shape"
   },
+  "BvOcQf": {
+    "note": "Custom colors",
+    "text": "Customised colours"
+  },
+  "Bzw++H": {
+    "note": "label for the remove animation button",
+    "text": "remove"
```
Comment (mukeshsahay) [inline packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json:272]: `Remove`

**Review by mukeshsahay — APPROVED** [review]
Added a few comments. Those should be checked and fixed in the original EN-US file.

File: packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json (hunk 5 above)
Comment (divya-sea) [reply packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json:272]: +1

File: packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json (hunk 4 above)
Comment (divya-sea) [reply packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json:344]: +1

File: packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json (hunk 3 above)
Comment (divya-sea) [reply packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json:468]: I think it is an aria label?

File: packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json [hunk 6]
```diff
@@ -699,10 +811,22 @@
...
+    "note": "Form field label",
+    "text": "Tag style"
+  },
   "kYpWHA": {
     "note": "placeholder",
     "text": "Select an option"
   },
+  "kdEHiD": {
+    "note": "Menu Item",
+    "text": "Remove all data binding"
```
[the remark this answers is not in the document]
Comment (divya-sea) [reply packages/scene-composer/translations/IotAppKitSceneComposer.en_UK.json:828]: +1

**Pull request merged**